config/prod.exs

```diff
@@ -226,6 +226,20 @@ config :archethic, ArchethicWeb.FaucetController,
 # which you should run after static files are built and
 # before starting your production server.
 config :archethic, ArchethicWeb.Endpoint,
+  explorer_url:
+    URI.to_string(%URI{
+      scheme: "https",
+      host:
+        case(System.get_env("ARCHETHIC_NETWORK_TYPE") == "testnet") do
+          true ->
+            System.get_env("ARCHETHIC_DOMAIN_NAME", "testnet.archethic.net")
+
+          false ->
+            System.get_env("ARCHETHIC_DOMAIN_NAME", "mainnet.archethic.net")
+        end,
+      port: System.get_env("ARCHETHIC_HTTPS_PORT", "50000") |> String.to_integer(),
```

Comment:
I think the port on mainnet.archethic.net and testnet.archethic.net would be the default ssl port 443. Port 50000 is used when querying directly the node with it's ip